pandas/tests/frame/methods/test_quantile.py

```diff
@@ -7,14 +7,29 @@
 
 
 class TestDataFrameQuantile:
-    def test_quantile_sparse(self):
+    @pytest.mark.parametrize(
+        "df,expected",
+        [
+            [
+                pd.DataFrame(
+                    {
+                        0: pd.Series(pd.arrays.SparseArray([1, 2])),
+                        1: pd.Series(pd.arrays.SparseArray([3, 4])),
+                    }
+                ),
+                pd.Series([1.5, 3.5], name=0.5),
+            ],
+            [
+                pd.DataFrame(pd.Series([0.0, None, 1.0, 2.0], dtype="Sparse[float]")),
+                pd.Series([1.0], name=0.5),
+            ],
+        ],
+    )
+    def test_quantile_sparse(self, df, expected):
         # GH#17198
-        s = pd.Series(pd.arrays.SparseArray([1, 2]))
-        s1 = pd.Series(pd.arrays.SparseArray([3, 4]))
-        df = pd.DataFrame({0: s, 1: s1})
-        result = df.quantile()
+        # GH#24600
+        result = df.quantile(0.5)
```

Comment:
hmm this should work with `df.quantile()` yes?

Reply:
I think we test this elsewhere so prob ok here

Reply:
Yes `df.quantile()` works the same. I can change this.